threads/main.go

```diff
@@ -0,0 +1,196 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"github.com/c-bata/go-prompt"
+	ma "github.com/multiformats/go-multiaddr"
+	"github.com/textileio/go-threads/api/client"
+	"github.com/textileio/go-threads/util"
+	"google.golang.org/grpc"
+	"log"
+	"os"
+	"strings"
+	"time"
+)
+
+var apiClient *client.Client
+var apiTimeout = time.Second * 5
+var streamTimeout = time.Second * 30
+var currentStore string
+
+type any map[string]interface{}
+
+var suggestions = []prompt.Suggest{
+	{
+		Text:        "use",
+		Description: "Switch the store.",
+	},
+	{
+		Text:        "exit",
+		Description: "Exit.",
+	},
+}
+
+var storeSuggestions = []prompt.Suggest{
+	{
+		Text:        "listen",
+		Description: "Stream all store updates.",
+	},
+	{
+		Text:        "getModel",
+		Description: "Get all models by model name.",
+	},
+}
+
+func completer(in prompt.Document) []prompt.Suggest {
+	w := in.GetWordBeforeCursor()
+	if w == "" {
+		return []prompt.Suggest{}
+	}
+	if currentStore != "" {
+		return prompt.FilterHasPrefix(append(suggestions, storeSuggestions...), w, true)
+	}
+	return prompt.FilterHasPrefix(suggestions, w, true)
```

Comment:
This just gives the use basic prompts when just started or extended prompts once they are using a store